src/Commands/Generators/NewShieldOauthGenerator.php

```diff
@@ -63,19 +64,41 @@ class NewShieldOauthGenerator extends BaseCommand
      * @var array<string, string>
      */
     protected $options = [
-        '-f' => 'Force overwrite existing file.',
+        '--force' => 'Force overwrite existing file.',
```

Comment:
Why did it change?

Reply:
The `GeneratorTrait` is already in use, so it is better to take advantage of it's built-in `--force` option instead of having to rewrite another one.